api/turing/api/request/request.go

```diff
@@ -22,12 +22,13 @@ type CreateOrUpdateRouterRequest struct {
 
 // RouterConfig defines the properties of the specific router version
 type RouterConfig struct {
-	Routes           models.Routes           `json:"routes" validate:"required"`
-	DefaultRouteID   *string                 `json:"default_route_id"`
-	TrafficRules     models.TrafficRules     `json:"rules" validate:"unique=Name,dive"`
-	ExperimentEngine *ExperimentEngineConfig `json:"experiment_engine" validate:"required,dive"`
-	ResourceRequest  *models.ResourceRequest `json:"resource_request"`
-	Timeout          string                  `json:"timeout" validate:"required"`
+	Routes            models.Routes             `json:"routes" validate:"required"`
+	DefaultRouteID    *string                   `json:"default_route_id"`
+	TrafficRules      models.TrafficRules       `json:"rules" validate:"unique=Name,dive"`
+	ExperimentEngine  *ExperimentEngineConfig   `json:"experiment_engine" validate:"required,dive"`
+	ResourceRequest   *models.ResourceRequest   `json:"resource_request"`
+	AutoscalingPolicy *models.AutoscalingPolicy `json:"autoscaling_policy" validate:"omitempty,dive"`
```

Comment:
`omitempty` is for backward compatibility. Can we removed when `getAutoscalingPolicyOrDefault` below is removed.

Reply:
any reason to remove the default? just thinking what if user don't know the difference, is this deliberate to force them to understand how they want to scale?

Reply:
The default is set by the clients - Turing UI and SDK, if the user doesn't explicitly supply or change these values. So, the API will expect them to be there.